### ISSUE TITLE
False negative unused variable with pre-increment

### ISSUE THREAD

**Issue by MilanPala** [slevomat/coding-standard#1173]
Since this commit https://github.com/slevomat/coding-standard/commit/d433062ee6be8bf2b0700f207f7426efc5cb7f73 I have issue with pre-increment in while loop. Sniff gives me error `Unused variable $i.` at line 3 but I think it's OK.

```
1. $i = 0;
2. while (...) {
3.	$x = $x . (++$i);
4. }
```

**Comment by kukulich** [comment]
@MilanPala Have you tried dev-master?

**Comment by MilanPala** [comment]
In master is it already fixed, issue is only in latest release. Good job, thank you, I will wait until next release.

**Comment by MilanPala** [comment]
Sorry for inconvenience, but the false positive is still in master, reproduced in PR https://github.com/slevomat/coding-standard/pull/1177

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/013cf66b0ebdcc6c8d7147be3571a84b89701967
